DESCRIPTION

```diff
@@ -0,0 +1,55 @@
+Type: Package
+Package: GAMBLR.viz
+Title: Make Plots for Genomic Analysis of Mature B-cell Lymphomas in R
+Version: 0.1.0
+Authors@R:
+    person("Vladimir", "Souza", , "vsouza@bcgsc.ca", role = c("aut", "cre"))
+Description: This package is part of the Genomic Analysis of Mature B-cell
+    Lymphomas (GAMBL) project developed by the Morin Lab.
+License: MIT + file LICENSE
+Depends:
+    R (>= 4.1)
```

Comment:
This should work on the 3.6.3 version too, right?

Reply:
To test the package with `R 3.6.3`, on `gphost01`, prior `GAMBLR.viz` installation, I tried to install `GAMBLR`, but I couldn't because of this dependency issue:
```
package ‘cBioPortalData’ is not available (for R version 3.6.3) 
```